syftlib/src/main/java/org/openmined/syft/WebRTC.kt

```diff
@@ -69,8 +74,10 @@ class WebRTCClient(
         peers.remove(newWorkerId)
     }
 
-    // Given a message, this function allows you to "broadcast" a message to all peers
-    // Alternatively, you may send a targeted message to one specific peer (specified by the "to" param)
+    /**
+     * @param message : this function allows you to "broadcast" this `message` to all peers
+     * @param to : you may send a targeted message to target `to`, default null
+     */
```

Comment:
If it defaults to `null` maybe the function could do it for us :)

Reply:
I didnt understand what you mean by this!

Reply:
`to: String? = null`

Reply:
Ahh Ok ok